odl/util/pytest_config.py

```diff
@@ -8,21 +8,30 @@
 
 """Test configuration file."""
 
-from __future__ import print_function, division, absolute_import
-import numpy as np
+from __future__ import absolute_import, division, print_function
```

Comment:
Unrelated, but python 2 is no longer supported by the community. We could remove support for it in ODL.

Reply:
Yes definitely. I'll make an issue for that. Perhaps it's okay to say "this is the last commit that supports Python 2", rather than crafting a release for it.